### ISSUE TITLE
Guide: Definition of "Community"

### ISSUE THREAD

**Issue by stuartasutton** [CredentialEngine/CredentialRegistry#117]
Throughout the Credential Engine Registry Guide, the term "Community" is used and yet nowhere that I've run across is that term defined. The examples throughout use "ce-registry" as "community name". Are there other communities of concern? In existence?

The Guide states:

>"The data is organized and stored in a community based metadata registry. Using this your community can have a full data store with an api, data validation and search capabilities, by just providing a simple config with a json-schema definition."

Who exactly is the "you" in "your community"?

**Comment by leolara** [comment]
@stuartasutton A community is a partition of the data, that has its own schemas and configuration and entity types. Currently we only have one community ce-registry, and it is treated as the default community. Hence, you should be able to ignore the community concept all together, and then you will be accessing the ce-registry community.

How can we explain this better?

**Comment by stuartasutton** [comment]
We discussed this yesterday on the teams' Tuesday call. I believe, others pipe in if I am wrong, that we reached the conclusion that since the CER was in fact the only community (of concern) that bringing up the notion was somewhat pointless. If we were concerned with more than one community, it would matter. My confusion stemmed from having it mentioned, not defined, and then never being used as though it mattered (at the level of the Guide). So, I think that references will be removed from the Guide.
